### DISCUSSION TITLE
Is Prowler App Free to Use

### DISCUSSION THREAD

**Discussion by vinayhr5** [prowler-cloud/prowler#6193] [Q&A] [answered]
Hi,

Quick question—Is the Prowler app free to use? It’s asking me to sign up and log in.
Which I've installed with docker compose as below

curl -LO https://raw.githubusercontent.com/prowler-cloud/prowler/refs/heads/master/docker-compose.yml
curl -LO https://raw.githubusercontent.com/prowler-cloud/prowler/refs/heads/master/.env
docker compose up -d

**Comment by jfagoagas** [comment, marked as answer]
Hello @vinayhr5, Prowler App is free to use with the on-premise installation https://docs.prowler.com/projects/prowler-open-source/en/latest/#prowler-app-installation. There is also a managed version that you can also use at https://cloud.prowler.com with a 15-day free trial and a free tier up to 10k unique scanned resources by month.

**Reply by vinayhr5** [reply]
Thanks
